wiki/geyser/playit-gg.md

```diff
@@ -23,15 +23,18 @@ The playit.gg Minecraft server plugin does not support UDP tunnels. You will nee
    ![img](/img/wiki/playit-gg/add_tunnel.png)
 4. Once "Add tunnel" is clicked, it should create a new tunnel, and you are set! Scroll down until you see this:
    ![img](/img/wiki/playit-gg/added_tunnel.png)
-   If you have Geyser running on a port that is not 19132, update the port there. The "Local Address" does not need to be changed unless you are not running playit.gg and Geyser on the same device.
-5. Connect to your server - use the IP and Port from the "Allocation" tab. In our example - "180.ip.ply.gg" as the IP, and "17019" as the port. Alternatively, use the Domain it gives you instead of the IP.
-6. If you join successfully, then you are done! Make sure to leave the playit.gg program running as closing it will close the tunnel. You also may want to ratelimit individual connections - use the "Per Connection Rate Limit" option to do so.
+   In Geyser config, set the `broadcast-port` to the playit.gg port from the "Allocation" tab. This will fix ping issues.
+   **Please do not change your Geyser port in `config.yml`** unless you have a reason to (e.g. hosting another Geyser server on the same machine), in which case skip to the last sentence. The bedrock (Geyser) port in `config.yml` and the playit.gg ports are entirely seperate, playit.gg will forward its port to the default Geyser port and should already work, changing it can cause errors. If you have changed your changed your config port, change the bedrock port back to the default of 19132 and ensure that `clone-remote-port=false`.
```

Comment:
```suggestion
   **Please do not change your Geyser port in `config.yml`** unless you have a reason to (e.g. hosting another Geyser server on the same machine), in which case skip to the last sentence. The bedrock (Geyser) port in `config.yml` and the playit.gg ports are entirely seperate, playit.gg will forward its port to the default Geyser port and should already work, changing it can cause errors. If you have changed your config port, change the bedrock port back to the default of 19132 and ensure that `clone-remote-port` is `false`.
```